lms/values.yaml

```diff
@@ -36,6 +36,7 @@ lms:
   replicaCount: 3
   releaseName: "molt-lms"
   sourceDialect: ""
+  targetDialect: "Cockroach"
```

Comment:
This should be in the orchestrator block not the lms block. Can you also lowercase the c?

Reply:
Ah right, good catch! Done.